<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>button1</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>391</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>371</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>button2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menubrowse">
    <property name="title">
     <string>browse</string>
    </property>
    <addaction name="actionsong"/>
    <addaction name="actionsong2"/>
   </widget>
   <addaction name="menubrowse"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionsong">
   <property name="text">
    <string>song1</string>
   </property>
  </action>
  <action name="actionsong2">
   <property name="text">
    <string>song2</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
